src/pages/Home.tsx

```diff
@@ -1,11 +1,184 @@
+    backgroundSize: "cover",
+    color: "#5b5b5b",
+    fontSize: "17px"
+  },
+  text: {
+    marginBottom: "15px",
+    fontWeight: 600
+  },
+  textHead: {
+    color: "white",
+    textAlign: "center",
+    width: "30vw",
+    fontWeight: 500,
+    fontSize: "16px",
+    lineHeight: "26px",
+    minWidth: "140px"
+  },
+  submit: {
+    textAlign: "center",
+    bottom: theme.spacing(5)
+  },
+  button: {
+    fontWeight: 400
+  },
+  title: {
+    marginTop: theme.spacing(2),
+    marginLeft: theme.spacing(3),
+    marginBottom: theme.spacing(1)
+  },
+  searchTool: {
+    marginBottom: theme.spacing(1)
+  },
+  modal: {
+    outline: "none"
+  }
+}));
 
 function Home() {
+  const styles = useStyles();
+  const [showNeedSubmitModal, setShowNeedSubmitModal] = useState(false);
+  const [showLeadSubmitModal, setShowLeadSubmitModal] = useState(false);
+  const [error, setError] = useState("");
+  const [mutateNeed] = useMutation(MUTATE_NEED);
+
+  const onNeedSubmit = (need: Need) => {
+    mutateNeed({
+      variables: { request: need }
+    })
+      .then(() => setShowNeedSubmitModal(false))
+      .catch(() => setError("Internal server error, please try again later"));
+  };
+
+  const onLeadSubmit = (lead: Lead) => {
+    mutateNeed({
+      variables: { request: lead }
+    })
+      .then(() => setShowLeadSubmitModal(false))
+      .catch(() => setError("Internal server error, please try again later"));
+  };
+
   return (
     <>
       <NavBar />
-      <h1>Home page</h1>
+      <Box className={styles.head}>
+        <Container className={styles.textHead}>
+          Aggregating global demand & supply of medical supplies & equipment to
+          connect needs with supply leads. STAT.
+        </Container>
+      </Box>
+      <Box className={styles.container}>
+        <Container className={styles.submitNeed}>
+          <Container className={styles.text}>
+            On the front-lines flighting COVID-19 and in need of medical
+            supplies & equipment?
+          </Container>
+          <Container className={styles.submit}>
+            <Button
+              onClick={() => setShowNeedSubmitModal(true)}
+              variant="contained"
+              color="primary"
+              className={styles.button}
+            >
+              POST NEEDS
+            </Button>
+          </Container>
+          <CreateOrEditNeedModal
+            title="Create Need"
+            onSubmit={onNeedSubmit}
+            submitError={error}
+            onClose={() => setShowNeedSubmitModal(false)}
+            isOpen={showNeedSubmitModal}
+          />
+        </Container>
+
+        <Container className={styles.submitLead}>
+          <Container className={styles.text}>
+            Have access to medical supplies & equipment needed by healthcare
+            professionals?
+          </Container>
+          <Container className={styles.submit}>
+            <Button
+              onClick={() => setShowLeadSubmitModal(true)}
```

Comment:
User needs to be authenticated to open this modal

Reply:
Talked with you about this on Slack, this will be implemented in a different ticket

Reply:
I created a new function that checks if user is authenticated, and if not will redirect to "/login"